graphene_django/debug/tests/test_query.py

```diff
@@ -24,7 +25,7 @@ class Meta:
 
     class Query(graphene.ObjectType):
         reporter = graphene.Field(ReporterType)
-        debug = graphene.Field(DjangoDebug, name="_debug")
+        debug = graphene.Field(DjangoDebug, name="__debug")
```

Comment:
This is just for consistency, most of the instances in this file were using double underscore.